### ISSUE TITLE
'npm install' fails for simple example

### ISSUE THREAD

**Issue by lowellk** [erikras/redux-form#2498]
I cannot run the simple example - I get an error from `npm install`. 

To repro:
- download fresh repo from github
- cd to `examples/simple`
- run `npm install`
- not the error below (I've tried this on a few Macs)

```
➜  simple git:(master) npm install
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts will run only for `npm publish`.
npm WARN prepublish-on-install (In npm@4 and previous versions, it also runs for `npm install`.)
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.

> redux-form@6.4.3 prepublish /Users/lowellk/workspace/redux-form
> npm run test && npm run clean && npm run build


> redux-form@6.4.3 test /Users/lowellk/workspace/redux-form
> mocha --compilers js:babel-register --recursive --recursive "src/**/__tests__/*" --require src/__tests__/setup.js

sh: mocha: command not found

npm ERR! Darwin 16.3.0
npm ERR! argv "/usr/local/Cellar/node/7.4.0/bin/node" "/usr/local/bin/npm" "run" "test"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! redux-form@6.4.3 test: `mocha --compilers js:babel-register --recursive --recursive "src/**/__tests__/*" --require src/__tests__/setup.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the redux-form@6.4.3 test script 'mocha --compilers js:babel-register --recursive --recursive "src/**/__tests__/*" --require src/__tests__/setup.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the redux-form package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     mocha --compilers js:babel-register --recursive --recursive "src/**/__tests__/*" --require src/__tests__/setup.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs redux-form
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls redux-form
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lowellk/workspace/redux-form/npm-debug.log

npm ERR! addLocal Could not install /Users/lowellk/workspace/redux-form
npm ERR! Darwin 16.3.0
npm ERR! argv "/usr/local/Cellar/node/7.4.0/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code ELIFECYCLE
npm ERR! redux-form@6.4.3 prepublish: `npm run test && npm run clean && npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the redux-form@6.4.3 prepublish script 'npm run test && npm run clean && npm run build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the redux-form package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run test && npm run clean && npm run build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs redux-form
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls redux-form
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lowellk/workspace/redux-form/examples/simple/npm-debug.log
```

Also, the following command fails: `npm run test && npm run clean && npm run build`

```
> @ test /Users/lowellk/workspace/redux-form/examples/simple
> echo 'Error: no test specified'

Error: no test specified

> @ clean /Users/lowellk/workspace/redux-form/examples/simple
> rimraf dist

sh: rimraf: command not found

npm ERR! Darwin 16.3.0
npm ERR! argv "/usr/local/Cellar/node/7.4.0/bin/node" "/usr/local/bin/npm" "run" "clean"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! @ clean: `rimraf dist`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the @ clean script 'rimraf dist'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rimraf dist
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs 
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lowellk/workspace/redux-form/examples/simple/npm-debug.log
```

**Comment by gustavohenke** [comment]
Because the examples contain relative paths to `redux-form` module, it seems like you hit this issue: 
npm/npm#15097 npm/npm#15114

Could you please try upgrading to npm v4.1.0, and see if it fixes the problem?

**Comment by lowellk** [comment]
Still getting the same error with npm 4 with both node 6 and 7

**Comment by lowellk** [comment]
By 'the same error', I'm referring to the first one I mentioned: `sh: mocha: command not found`

**Comment by lejema160** [comment]
I have the same error ! 
I tried to install mocha it does not work.

Node version : v6.9.1
Npm version : 4.1.2

```
C:\Downloads\redux-form-master\examples\simple>npm install
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts will run only for `npm publish`.
npm WARN prepublish-on-install (In npm@4 and previous versions, it also runs for `npm install`.)
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.

> redux-form@6.4.3 prepublish C:\Downloads\redux-form-master
> npm run test && npm run clean && npm run build


> redux-form@6.4.3 test C:\Downloads\redux-form-master
> mocha --compilers js:babel-register --recursive --recursive "src/**/__tests__/*" --require src/__tests__/setup.js

'mocha' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "test"
npm ERR! node v6.9.1
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! redux-form@6.4.3 test: `mocha --compilers js:babel-register --recursive --recursive "src/**/__tests__/*" --require src/__tests__/setup.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the redux-form@6.4.3 test script 'mocha --compilers js:babel-register --recursive --recursive "src/**/__tests__/*" --require src/__tests__/setup.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the redux-form package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     mocha --compilers js:babel-register --recursive --recursive "src/**/__tests__/*" --require src/__tests__/setup.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs redux-form
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls redux-form
npm ERR! There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Downloads\redux-form-master\npm-debug.log

npm ERR! addLocal Could not install C:\Downloads\redux-form-master
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v6.9.1
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! redux-form@6.4.3 prepublish: `npm run test && npm run clean && npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the redux-form@6.4.3 prepublish script 'npm run test && npm run clean && npm run build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the redux-form package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run test && npm run clean && npm run build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs redux-form
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls redux-form
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Downloads\redux-form-master\examples\simple\npm-debug.log
```

**Comment by lejema160** [comment]
@lowellk did you find the problem ?

**Comment by matiaslee** [comment]
Hi people,

I had the same problem. After a while, I restarted from scratch and I noticed the documentation says: 

"To run this example locally on your machine:
1 - Clone the redux-form repository, then 
2 - cd redux-form to change to the repo directory, and 
3 - run: npm install.

Then run npm run example:simple or manually run the following commands:

cd ./examples/simple
npm install
npm start" 

I was missing the steps 1,2,3. Best!
